docs/configuration_file.rst

```diff
@@ -1,22 +1,1167 @@
+instead of using the convex hull of well perforations. Currently requires an input reservoir simulation model. 
+
+.. _prior_volume_distribution:
+
+prior_volume_distribution
+-------------------------
+
+Volume distribution method of tubes (or cells in tube) to be applied on the prior volume distribution. 
+Based on tube length by default.
+
+Valid options are:
+
+* tube_length: distrubutes the volume of the convex hull of the FlowNet model,
+  based on the length of a tube. I.e., if all tubes have equeal lenght, they
+  will have equal volume.
+* voronoi_per_tube: distributes the input models bulk volume of active cells
+  to the nearest FlowNet tube of a cell. The total volume of the tube is then
+  devided equally over the cells of the tube. I.e., in areas with a higher
+  FlowNet tube density, the volume per cell is lower. Mind that if the FlowNet
+  model, i.e., the convex hull of the well connections, is much smaller than the
+  original model volume outside of the well connection convex hull might be
+  collapsed at the borders of the model. I.e., the borders of your model could
+  get unrealisticly large volumes. This can be mitigated by increasing the hull
+  factor of the FlowNet model generation process or by setting the
+  place_nodes_in_volume_reservoir to true.
+
+prod_control_mode
+-----------------
+
+Defines how the production wells are controlled in the historic production period. Available modes are *ORAT*, *GRAT*, *WRAT*, *LRAT*, *RESV*, *BHP*.
+
+pvt
+---
+
+rsvd
+~~~~
+
+The path to a csv  file with RSVD input. This file can now be done either as one table used for all EQLNUM regions, 
+or as one table for each EQLNUM region. The csv file needs a header with column names "depth", "rs" and "eqlnum" 
+(the latter only when multiple tables are defined).
+
+norne_static/rsvd_multiple.csv
+
+random_seed
+-----------
+
+An integer. Set this to control the numpy random number generator, to make sure that your FlowNet models are possible to regenerate 
+(meaning that two FlowNet runs with the exact same input config file will produce the same FlowNet model).
+
+training_set_end_date
+---------------------
+
+The last date to be used for conditioning/training of the FlowNet network model. The date of course 
+needs to be within the date range of the observations provided in the input data.
+
+Defining this at the same time as **training_set_fraction** will raise a ValueError.
+
+
+training_set_fraction
+---------------------
+
+A number between 0 and 1 defining how much of the input data should be used for conditioning/training of 
+the FlowNet network model. If there are 10 years of input obervations of e.g. WOPR, a *training_set_fraction*
+of 0.6 will use 6 years of the input data for training (leaving 4 years of data for validation).
+
+Defining this at the same time as **training_set_end_date** will raise a ValueError.
+      
+Example of the entire flownet part of the configuration yaml file:
```

Comment:
Perhaps this complete example could be moved to the very top, in the introduction to flownet?